chiselFrontend/src/main/scala/chisel3/core/Bits.scala

```diff
@@ -1020,3 +1021,51 @@ object FixedPoint {
   }
 
 }
+
+/** Data type for representing bidirectional bitvectors of a given width
```

Comment:
Do you want to limit this to bitvectors, or could this be a way to thread actual analog signals through Chisel? The more generalized definition of width would just be (parallel) wires in an analog bus.